vcd/datasource_vcd_external_network.go

```diff
@@ -6,7 +6,8 @@ import (
 
 func datasourceVcdExternalNetwork() *schema.Resource {
 	return &schema.Resource{
-		Read: resourceVcdExternalNetworkRead,
+		Read:               resourceVcdExternalNetworkRead,
```

Comment:
Should we add a `.changes/v3.8.0/903-deprecations.md` mention or is it not needed?
(Same question for adding a note in `website/docs/d/external_network.html.markdown`, not sure if it's worth warning there also)

Reply:
Good point. Added changelog note and warning messages for the `vcd_external_network`